www/blog/2021/02/11/megacities/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: Megacities
+published: false
+date: 2021-02-11
+permalink: /blog/2021/02/11/megacities/
+category: blog
+excerpt: "Building on the global locality coverage in Who's On, we've updated our megacities."
+authors: [stepps00]
+image: "images/jakarta.jpg"
+tag: [megacity,locality,whosonfirst,wof,data]
+---
+
+<img src="images/jakarta.jpg" style="max-height:none !important;" />
+
+## Updating our Megacity Localities
+
+In 2019, Who's On First announced the addition of comprehensive global coverage at the locality placetype; read more about that work [here](https://www.whosonfirst.org/blog/2019/05/13/geonames/). Building on this locality work, we've recently reviewed some of these more populous localities, adding hand-curated geometries, new name translations, and concordances.
+
+## Megacities
+
+Represented by the [`wof:megacity`](https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof/megacity.json) property, these Who’s On First records represent the world's most populated cities. According to [Wikipedia](https://en.wikipedia.org/wiki/Megacity), a "megacity is a very large city, typically with a population of more than 10 million people." Other sources suggest megacities are defined as having populations lower, around 5 million. While Who's On First does not have a population cut-off for how these megacities are defined, we have relied on Natural Earth's [Populated Places](http://www.naturalearthdata.com/downloads/10m-cultural-vectors/) file for this work. Natural Earth identifies 590 global populated places, each identified with a `MEGACITY` attribute, name translations, and concordances.
```

Comment:
Let's document the vintage and population cutoff from Natural Earth.

> Natural Earth identifies 590 global populated places, each identified with a `MEGACITY` attribute (actual or emergent), name translations, and concordances, for urban agglomerations with a population of 750,000 or more in 2010.

From:

> Natural Earth identifies 590 global populated places, each identified with a `MEGACITY` attribute, name translations, and concordances.

Reply:
addressed in ed9a9fe.